include/pybind11/async.h

```diff
@@ -0,0 +1,142 @@
+#pragma once
+
+#include <memory>
+#include <future>
+#include <map>
+#include <algorithm>
+#include <sstream>
+
+#include <chrono>
+#include <iostream>
```

Comment:
Do we need all these? At least `<iostream>` and `<sstream>` are definitely not used, right?

Reply:
you are right, i just picked everything from where i had it before :) still have to clean up...